pkg/network/node/ovscontroller.go

```diff
@@ -58,10 +58,10 @@ func (oc *ovsController) AlreadySetUp() bool {
 		return false
 	}
 	expectedVersionNote := oc.getVersionNote()
-	for _, flow := range flows {
-		parsed, err := ovs.ParseFlow(ovs.ParseForDump, flow)
-		if err == nil && parsed.Table == ruleVersionTable && parsed.NoteHasPrefix(expectedVersionNote) {
-			return true
+	for i := len(flows)-1; i >= 0; i-- {
+		parsed, err := ovs.ParseFlow(ovs.ParseForDump, flows[i])
```

Comment:
Is there a go bindings for ovs that we can use instead of calling ovs CLI commands?

Reply:
No. There are bindings for the ovs-vsctl side of things, but not ovs-ofctl.

But this patch was mostly just a quick fix for 3.7.z. I have ideas on further improvements already.